### DISCUSSION TITLE
Comings, Goings, and New Arrivals

### DISCUSSION THREAD

**Discussion by QuintillusCFC** [C7-Game/Prototype#251] [General]
Similar to the CFC OT thread, a thread for posting about known absences or new arrivals.

With the arrival of spring, I'm spending much more time on non-Civ-related and non-coding-related hobbies, including a lot of travel.  As a result, my involvement is going to be much lower through the end of Q2 than it was this winter.  I might occasionally resurface on rainy days when I'm not traveling, but largely will be busy taking advantage of the analog, outdoorsy parts of life.

This has been the case for a couple weeks already, and I'd met to post it then, but after the groundhog's 9 more weeks of winter, spring snuck up on me.

For any PRs or other things in progress related to me, feel free to resolve them as appropriate without awaiting my return.

**Comment by WildWeazel** [comment]
**Thank you** for everything that you've done for C7. We wouldn't be anywhere near Carthage by now without you taking on the project management. We'll keep the wheels turning in your absence. I saw that you hadn't been active online for a while and figured you were taking another AFK break. It's already HOT here, when outdoorsy stuff actually winds down for summer. I'll be local again in a few weeks if you're around and up for lunch again.

If you have the time, I'd appreciate any light you can shed on my struggles with VSStub and .NET 6 in #246

**Reply by QuintillusCFC** [reply]
Already hot?  Wow!  I was just in North Carolina, and it was kind of chilly down there.  Actually downright cold on Sunday, I nearly froze because I didn't expect Ohio-late-February weather in the South in May.  But other than Sunday, it was pleasant during the day, in the low-to-mid-70s, if chilly in the mornings.

Shoot me a message when you'll be local again, beyond next week I don't know exactly when I'll be where, but I may well be around.  Next week, I might have time to look at #246... alas I'm heading to sleep very soon tonight, since I have a lot planned for tomorrow.

**Comment by JimOfLeisure** [comment]
I've also been mostly disengaged from the project for a while. I've been peeking in a lot, but my dev interests are elsewhere lately (web dev). I guess at some point there may be an interest in replacing the build Docker image with a dotnet-based one instead of mono-based, and I might poke at that, but I don't see myself diving back into the game itself in the near future.

**Comment by maxpetul** [comment]
I haven't gone anywhere but I also haven't contributed much code in the past few weeks. Not for any special reason, just my interest naturally lead me back to C3X. I haven't seen much of @pcen recently, it's interesting how we all ended up taking a break at about the same time.

Since the project's in a lull, now would be a good time to find and tabify the many files that need it. We should also standardize formatting and capitalization while we're at it. I thought of this back during the lull around Christmas but never got around to it. Though it's probably easiest to first merge the waiting PRs so as not to conflict with them. I know that there's at least nonstandard formatting & capitalization in my Combat Part Two PR.

**Comment by pcen** [comment]
Yes I've also been taking a break recently, I've been in the process of moving and starting a new internship, so I haven't had any time to contribute to C7. I plan on coming back at some point in the relatively near future once I have more free time.

**Comment by WildWeazel** [comment]
With everyone taking a break, I should take the lull in activity as an opportunity to do some architectural refactoring and cleanup.

**Comment by QuintillusCFC** [comment]
I'm going to be away from the project from July 31st to around the middle of August.  Traveling to a new-to-me corner of North America, the Northeast, over the course of two weeks.  Feel free to pick up any paused PRs during that time!

I didn't actually expect to work on the project as much as I did in July, but it was hot and humid here.  Thus why I'm happy that I'll be traveling north rather than south!

**Comment by QuintillusCFC** [comment]
So just as @pcen happened to swing back around, I was concluding no one else was working on this project, and there weren't signs of that changing any time soon, and thus decided to take on another project with an IRL friend.  That means my time available for this project will be low for the next half year or so.

**Comment by WildWeazel** [comment]
I forgot about this thread and don't know if they're even aware of the discussions section, but a belated welcome to the team @esbudylin and @TomWerner ! They helped get the Godot 4 port wrapped up and Carthage out the door.
